code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -7,23 +7,32 @@
 
 
 
-/mob/proc/bee_friendly()
-	return 0
+/mob/proc/bee_friendly() //Because we, are, your friends
```

Comment:
Have an actual doc comment instead. I honestly thought this was a pun at first.

Reply:
it's a song lyric